packages/firebase/compat/rollup.config.js

```diff
@@ -243,7 +243,7 @@ const completeBuilds = [
     plugins: [
       sourcemaps(),
       resolveModule({
-        mainFields: ['lite-esm5', 'esm5', 'module']
+        exportConditions: ['liteesm5', 'esm5']
```

Comment:
Is `liteesm5` missing a dash here?

Reply:
no, it's intentional to not include a dash here because it seems rollup doesn't like it.